src/__tests__/3box.test.js

```diff
@@ -61,10 +62,15 @@ jest.mock('../utils', () => {
           addressMap[did] = hash
           return { status: 'success', data: { hash } }
         case 'link': // make a link
-          did = payload.consent_msg.split(',')[1]
-          const address = payload.consent_signature.split(',')[1]
-          linkmap[address] = did
-          return { status: 'success', data: { did, address } }
+          if (firstLink) {
+            firstLink = false
+            return Promise.reject('{ status: "error", message: "an error" }')
```

Comment:
Is this error message used internally only? Would it make more sense if the message is more descriptive?

Reply:
This is just the mock. The real error would come from the address-server, but for our testing purposes it doesn't really matter what it says.